### ISSUE TITLE
error msg

### ISSUE THREAD

**Issue by cdrt1** [Genymobile/gnirehtet#407]
hi. fist time run after installation worked perfectly fine (the key did not appear though).
after disconnecting and closing the command prompt it is not running anymore and here is the error message:

C:\gnirehtet>gnirehtet run
2021-09-14 22:58:56.473 INFO Main: Starting relay server on port 31416...
2021-09-14 22:58:56.473 INFO Main: Checking gnirehtet client...
2021-09-14 22:58:56.473 INFO Relay: Relay server started
2021-09-14 22:58:57.541 **ERROR** Main: Cannot start client: Command adb ["shell", "
dumpsys", "package", "com.genymobile.gnirehtet"] returned with value 1

laptop OS: Windows 8.1
mobile OS: Android 10

I appreciate any comment / help.
cdr

**Comment by rom1v** [comment]
Check the output of `adb devices`: is your device correctly detected?

**Comment by cdrt1** [comment]
Thank you so much Romain,

You are right, the problem was that I changed the name of the mobile device
after gnirehtet install and it caused this problem.

I tried to download the files again and reinstall it again but this time I
get another error message:

C:\gnirehtet>gnirehtet install
2021-09-15 11:21:51.713 INFO Main: Installing gnirehtet client...
* daemon not running; starting now at tcp:5037
* daemon started successfully
Performing Push Install
adb: error: failed to get feature set: no devices/emulators found
2021-09-15 11:21:52.791 ERROR Main: Execution error: Command adb
["install", "-r
", "gnirehtet.apk"] returned with value -1

When I look at the registry I see "c:\gnirehtet" with Hex value = 0 only.
No other entries for gnirehte.

Any suggestions?
Thanks
cdr
